cicd/jenkins/openshift-jenkins.adoc

```diff
@@ -6,9 +6,11 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-{product-title} 4.11 moves the `OpenShift Jenkins` and `OpenShift Agent Base` images to the `ocp-tools-4` repository at `registry.redhat.io` so that Red Hat can produce and update the images outside the {product-title} lifecycle. Previously, these images were in the {product-title} install payload and the `openshift4` repository at `registry.redhat.io`.
+{product-title} 4.11 moves the "OpenShift Jenkins" and "OpenShift Agent Base" images to the `ocp-tools-4` repository at `registry.redhat.io`. It also removes the "OpenShift Jenkins Maven" and "NodeJS Agent" images from its payload:
```

Comment:
I added this abstract to meet the checklist requirement of providing a **single**-paragraph TLDR. Then I converted the two following paragraphs into bullets, which elaborate on the information in the abstract.